Javascript/homework/responses/4.1-page-setup.md

```diff
@@ -0,0 +1,25 @@
+### Week 4 - Toggling between digital and analog clocks and incoporating advanced features
```

Comment:
Make sure all the response files are formatted like described under "The Github Action" https://github.com/bitprj/BitCamp/tree/master/repo_logistics